src/components/Tabs/TabPanel.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { useTabsContext } from "./tabs-context";
+
+interface TabPanelProps {
+	id: string;
+	children: React.ReactNode;
+}
+
+export const TabPanel = ({ id, children }: TabPanelProps) => {
```

Comment:
Delete this; Can do this conditional in parent and provides greater flexibility to do it this way.